package.json

```diff
@@ -34,33 +34,12 @@
     "@babel/preset-env": "7.3.4",
     "@babel/preset-react": "7.0.0",
     "@babel/runtime": "7.3.4",
-    "@wordpress/a11y": "2.0.2",
-    "@wordpress/api-fetch": "2.2.8",
```

Comment:
Since a lot of these are used by Jetpack blocks, we need to test those to make sure they're still working (which they should, since their dependencies are in `packages/jetpack-blocks/package.json`. But better safe than sorry :sweat_smile: )

Reply:
I added the deps back. We can handle their removal in an other PR